_layouts/docwithnav.html

```diff
@@ -83,9 +83,14 @@
 	</div> <!-- /docsToc -->
 	<div id="docsContent" style="float:none; display:table-cell; width:auto; max-width:8in;">
 		{% if page.version == "master"  %}
-			<div style="color: #8a6d3b; background-color: #fcf8e3; border-color: #faebcc; padding: 15px; margin-bottom: 20px; border: 1px solid transparent; border-radius: 4px;"><p><b>Warning: You are viewing Master documentation. <a href="/v1.6">Click Here</a> For the latest release.</b></p></div>
+			<div style="text-align: center; color: #8a6d3b; background-color: #fcf8e3; border-color: #faebcc; padding: 15px; margin-bottom: 20px; border: 1px solid transparent; border-radius: 4px;">
+				<p><b>Warning: You are viewing Master documentation. <a href="/v1.6">Click Here</a> For the latest release.</b></p>
```

Comment:
1. it would look nice to have `master` as a code block
2. shouldn't master be lowercase?